packages/api/src/derives/types.ts

```diff
@@ -0,0 +1,20 @@
+import {RewardDestination} from '@cennznet/types';
+import {AccountId, EraIndex, Exposure, Keys, StakingLedger, ValidatorPrefs} from '@polkadot/types/interfaces';
+
+export interface DerivedStakingInfo {
```

Comment:
Do we always group all types files inside types folder, or any difference on this one?

Reply:
Its good to group all types in a file cause else api think its part of its derive